app/models/manageiq/providers/vmware/infra_manager/vm_or_template_shared/scanning.rb

```diff
@@ -17,10 +17,6 @@ def perform_metadata_scan(ost)
     vm_name  = File.uri_to_local_path(ost.args[0])
     $log.debug "#{log_pref} VM = #{vm_name}"
 
-    args1 = ost.args[1]
-    args1['ems'][:use_vim_broker]      = MiqServer.use_broker_for_embedded_proxy?(args1['ems']['connect_to'])
-    args1['ems'][:vim_broker_drb_port] = MiqVimBrokerWorker.drb_port if args1['ems'][:use_vim_broker]
```

Comment:
@yrudman I think using `args1` is shorthand for modifying `ost.args[1]` before passing it to `connect_to_ems(ost)`
Can you check if `ost.args[1]` has those keys set?